src/fontra/views/editor/panel-designspace-navigation.js

```diff
@@ -671,7 +681,12 @@ export default class DesignspaceNavigationPanel extends Panel {
       sourceController.addKeyListener("active", async (event) => {
         await this.sceneController.editGlyphAndRecordChanges((glyph) => {
           glyph.sources[index].inactive = !event.newValue;
-          return `${event.newValue ? "" : "de"}activate ${source.name}`;
+          return event.newValue
```

Comment:
Same, pls put the switch inside the translate() call.

Reply:
thanks. done.